src/main/java/seedu/guest/model/guest/DateRange.java

```diff
@@ -15,9 +15,9 @@
 public class DateRange {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "DateRanges should contain only numbers, hyphens, forward slashes, and spaces.\n"
-                    + "They must follow the format \"dd/MM/yy - dd/MM/yy\" and the second date "
-                    + "must be later than the first date.";
+            "DateRanges should contain only numbers, spaces, hyphens (-), and forward slashes (/).\n"
```

Comment:
Should the hyphen be expressed in singular form since we only accept 1 hyphen?

Reply:
Agreed, changed!